src/microbe_stage/organelle_components/ExternallyPositionedComponent.cs

```diff
@@ -65,6 +59,22 @@ public virtual void OnShapeParentChanged(Microbe newShapeParent, Vector3 offset)
     {
     }
 
+    /// <summary>
+    ///  Gets the angle of rotation of an externally placed organelle
+    /// </summary>
+    /// <param name="delta"> the difference between the cell middle and the external organelle position</param>
+    protected float GetAngle(Vector3 delta)
+    {
+        float angle = Mathf.Atan2(-delta.z, delta.x);
+        if (angle < 0)
+        {
+            angle = angle + (2 * Mathf.Pi);
+        }
+
+        angle = (angle * 180 / Mathf.Pi - 90) % 360;
+        return angle;
+    }
```

Comment:
This looks like a general purpose method that can be put inside a static helper class.

Reply:
Its only usefull for ExternalOrganelles, so i dont see the reason for an helper class.